packages/account/src/Sections/Security/ClosingAccount/closing-account-reason-form.tsx

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+import { Field, Formik, FormikErrors, FormikValues } from 'formik';
+import { Checkbox, FormSubmitButton, Input, Text } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+
+type TFormValues = {
+    'financial-priorities': boolean;
+    'stop-trading': boolean;
+    'not-interested': boolean;
+    'another-website': boolean;
+    'not-user-friendly': boolean;
+    'difficult-transactions': boolean;
+    'lack-of-features': boolean;
+    'unsatisfactory-service': boolean;
+    'other-reasons': boolean;
+    other_trading_platforms: string;
+    do_to_improve: string;
+};
+const initial_form: TFormValues = {
+    'financial-priorities': false,
+    'stop-trading': false,
+    'not-interested': false,
+    'another-website': false,
+    'not-user-friendly': false,
+    'difficult-transactions': false,
+    'lack-of-features': false,
+    'unsatisfactory-service': false,
+    'other-reasons': false,
+    other_trading_platforms: '',
+    do_to_improve: '',
+};
+type TClosingAccountReasonFormProps = {
+    validateFields: (values: FormikValues) => FormikErrors<FormikValues>;
+    onSubmit: (values: FormikValues) => void;
+    is_checkbox_disabled: boolean;
+    onChangeCheckbox: (
+        values: FormikValues,
+        field_name: keyof TFormValues,
+        setFieldValue: (name: string, values: string | boolean) => void
+    ) => void;
+    character_limit_no: number;
+    onInputChange: (
+        e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+        old_value: string,
+        onChange: (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => void
+    ) => void;
+    onInputPaste: (e: React.ClipboardEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+    remaining_characters: number;
+    onBackClick: () => void;
+};
+const ClosingAccountReasonForm = ({
+    validateFields,
+    onSubmit,
+    is_checkbox_disabled,
+    onChangeCheckbox,
+    character_limit_no,
+    onInputChange,
+    onInputPaste,
+    remaining_characters,
+    onBackClick,
+}: TClosingAccountReasonFormProps) => (
+    <Formik initialValues={initial_form} validate={validateFields} onSubmit={onSubmit}>
+        {({ values, setFieldValue, errors, handleChange, handleSubmit, dirty }) => (
+            <form onSubmit={handleSubmit}>
+                <Field name='financial-priorities'>
+                    {({ field }: FormikValues) => (
```

Comment:
```suggestion
{({ field }: FieldProps) => (
```

Reply:
As these checkboxes are all same in structure, and the only props which are changing are name & label
We can make an array of objects with name and label and loop over it

And we don't need to handle checkbox separately, Formik can handle them by itself
Ref: https://formik.org/docs/examples/checkboxes